frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { expandable, sortable, cellWidth } from '@patternfly/react-table';
+import {
+  StateCounts,
+  Severity,
+  AlertState,
+  alertsRowFilter,
+} from '@console/internal/components/monitoring';
+import { Kebab } from '@console/internal/components/utils';
+import { alertDescription, alertState, AlertSeverity } from '@console/internal/reducers/monitoring';
+import { Alert, Rule } from '@console/internal/components/monitoring/types';
+
+const viewAlertRule = {
+  label: 'View Alerting Rule',
+};
+
+const alertSeverityReducer = (alert: Alert) => alert?.labels?.severity;
+
+const alertSeverities = [
+  { id: AlertSeverity.Critical, title: 'Critical' },
+  { id: AlertSeverity.Warning, title: 'Warning' },
+  { id: AlertSeverity.Info, title: 'Info' },
+];
+
+export const monitoringAlertColumn = [
+  {
+    title: 'Name',
+    cellFormatters: [expandable],
```

Comment:
Rules that are not active (no firing alerts) have nothing to expand, so I think we should not be adding `expandable` in that case.

Reply:
Done. adding the expandable icon for rules which are in firing state.

Reply:
👍